### PR TITLE
Multi page layout

### PR DESCRIPTION
**Pull request by wagenrace**
I changed the layout of the site have muliple pages.

The text is all copied except contacts

### CODE REVIEW

**Comment by fheigwer** [comment]
Could You address my comment on Lassis changes to the membership statement on this nice looking Multi page revamp.

**Comment by fheigwer** [comment]
Could you also make the background less depressingly grey. What do you think about some nice looking microscopy imagery or so? Maybe you can check the 2019 hackathon page again. I still like its background.

**Comment by koalive** [comment]
Really nice! I agree with everyone else's comments.

In the "About" section, there's now a dead link to the "Resources" page.

> CytoData will compile useful resources to help disseminate technical and scientific advances in the field. Details are provided in a dedicated section.

And typos in the contact section:

> Our board member are spread over multiple continuations.

Maybe "Our board members are spread over multiple organizations and continents, so the CytoData society does NOT have a physical address or location."?

**Comment by wagenrace** [comment]
@fheigwer I have changed the background back to the old color, but will make mock-ups for a new style in later pr. The words I have not changed yet, I can not find the right ones :P 

 @koalive Thanks for your feedback, I have changed it.

**Comment by lopaavol** [comment]
Here are the comments I sent to the email thread:
- I would change the "About" page to "Society" as it includes information mostly about the basis of the society
- Should "Symposiums" be "Symposia"?
- In the Symposiums page, the last year event was orgainzed by Recursion, not rxrx.ai. This year is bit.bio, not bit.io.

**Comment by wagenrace** [comment]
> 
> 
> Here are the comments I sent to the email thread:
> 
>     * I would change the "About" page to "Society" as it includes information mostly about the basis of the society
> 
>     * Should "Symposiums" be "Symposia"?
> 
>     * In the Symposiums page, the last year event was orgainzed by Recursion, not rxrx.ai. This year is bit.bio, not bit.io.

- I would like to keep the 'About' given it is more commonly used on sites
- Symposiums and symposia both seem correct (https://en.wiktionary.org/wiki/symposium)
- Fixed the typos

**Comment by fheigwer** [comment]
@wagenrace I opt for "Symposia"

**Comment by fheigwer** [comment]
I think "About" should be "Society", too. In About I would expect some legal text about the page or so. It doesn't really fit the context.

File: content/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "CytoData Society"
+---
+
+![CytoData banner](/./main_files/cytodata-banner.png)
+The CytoData society (CytoDS) builds and maintains an active community around image-based profiling of biological phenotypes induced by genetic, chemical or other perturbations of biological systems.
+
+CytoData Society is an unregistered society meaning that the society does not have an official status in any country. Instead we are an active community of researchers and developers. We consider everyone in our mailing list as a member of the society.
```
Comment (fheigwer) [inline content/_index.md:8]: Move to bottom of page and give it a \<div style="background:#808080">\</div>

File: content/_index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "CytoData Society"
+---
+
+![CytoData banner](/./main_files/cytodata-banner.png)
```
Comment (fheigwer) [inline content/_index.md:5]: add "The field of image-based profiling is still in its infancy, and its adoption has been slow, primarily due to the data analysis challenges posed, the large imaging experiments required, and the lack of ground-truth data sets where relationships among images are known. The CytoData Society aims to bring image-based profiling into the mainstream by organizing a striving community working on this technology."

File: content/_index.md (hunk 1 above)
Comment (wagenrace) [reply content/_index.md:8]: Do you mean below the subscriptions button?

Comment (fheigwer) [reply content/_index.md:8]: yes

Comment (wagenrace) [reply content/_index.md]: I did test some thing with a darker background but nothing was nice. Italic seems to have the cleanest but still clear effect

Comment (fheigwer) [reply content/_index.md]: also good

**Pull request merged**